osf/models/notification_type.py

```diff
@@ -115,30 +108,23 @@ class Type(str, Enum):
         ADDON_FILE_COPIED = 'addon_file_copied'
         ADDON_FILE_RENAMED = 'addon_file_renamed'
         ADDON_FILE_MOVED = 'addon_file_moved'
-        ADDON_FILE_REMOVED = 'addon_file_removed'  # unused
+        ADDON_FILE_REMOVED = 'addon_file_removed'
         FOLDER_CREATED = 'folder_created'
 
         # Provider notifications
         PROVIDER_NEW_PENDING_SUBMISSIONS = 'provider_new_pending_submissions'
         PROVIDER_NEW_PENDING_WITHDRAW_REQUESTS = 'provider_new_pending_withdraw_requests'
         PROVIDER_REVIEWS_SUBMISSION_CONFIRMATION = 'provider_reviews_submission_confirmation'
-        PROVIDER_REVIEWS_MODERATOR_SUBMISSION_CONFIRMATION = 'provider_reviews_moderator_submission_confirmation'  # unused
-        PROVIDER_REVIEWS_REJECT_CONFIRMATION = 'provider_reviews_reject_confirmation'  # unused
-        PROVIDER_REVIEWS_ACCEPT_CONFIRMATION = 'provider_reviews_accept_confirmation'  # unused
         PROVIDER_REVIEWS_RESUBMISSION_CONFIRMATION = 'provider_reviews_resubmission_confirmation'
-        PROVIDER_REVIEWS_COMMENT_EDITED = 'provider_reviews_comment_edited'  # unused
-        PROVIDER_CONTRIBUTOR_ADDED_PREPRINT = 'provider_contributor_added_preprint'  # unused
         PROVIDER_CONFIRM_EMAIL_MODERATION = 'provider_confirm_email_moderation'
         PROVIDER_MODERATOR_ADDED = 'provider_moderator_added'
-        PROVIDER_CONFIRM_EMAIL_PREPRINTS = 'provider_confirm_email_preprints'  # unused
         PROVIDER_USER_INVITE_PREPRINT = 'provider_user_invite_preprint'
 
         # Preprint notifications
         PREPRINT_REQUEST_WITHDRAWAL_APPROVED = 'preprint_request_withdrawal_approved'
         PREPRINT_REQUEST_WITHDRAWAL_DECLINED = 'preprint_request_withdrawal_declined'
         PREPRINT_CONTRIBUTOR_ADDED_PREPRINT_NODE_FROM_OSF = 'preprint_contributor_added_preprint_node_from_osf'
         PREPRINT_CONTRIBUTOR_ADDED_DEFAULT = 'preprint_contributor_added_default'
-        PREPRINT_PENDING_RETRACTION_ADMIN = 'preprint_pending_retraction_admin'  # unused
```

Comment:
Maybe there should be something called `PREPRINT_REQUEST_WITHDRAWAL_PENDING`. Please take a look.